postfixadmin/stack.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+
+services:
+  db:
+    image: mysql:5.7
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: notSecureChangeMe
```

Comment:
You might consider swapping to simply using `MYSQL_RANDOM_ROOT_PASSWORD: 1` since you create an explicit non-`root` user/database (which makes access to the `root` account mostly unnecessary).